### DISCUSSION TITLE
Enable old style miniplayer in Youtube Music Revanced Extended

### DISCUSSION THREAD

**Discussion by MrKK4** [anddea/revanced-patches#387] [Q&A] [answered]
Hey guys, I saw a patch 
"YT Music - Enable old style miniplayer: Update description ([dee3a45](https://github.com/anddea/revanced-patches/commit/dee3a453b173fab46fb3c1c98026309e5e232bea))"

what does it do?
I'm asking cause I don't see any difference in my mini player
Is there anything I have to enable in settings
can anyone tell me
The old mini player that I know of is this
![image](https://github.com/anddea/revanced-patches/assets/94269452/7bc4731b-d4bb-425e-a6a6-edb427758484)

But the one which I have
![image](https://github.com/anddea/revanced-patches/assets/94269452/ac3ac5fd-7ce6-4ab2-9aff-a13b0ad5be30)
I belive this is the new style mini player.

so am i missing something?

**Comment by anddea** [comment, marked as answer]
Old style miniplayer is available only for older versions, because Google deleted code related to that. The commit/patch you mentioned just adds description that it is deprecated now. Old style miniplayer has next button and you can swipe down to close it, I guess nothing else other than that.

**Reply by MrKK4** [reply]
Oh! so I miss interpreted that patch note. Thanks for clarifying.
> Old style miniplayer has next button and you can swipe down to close it, I guess nothing else other than that.

Yeah I like those settings that's why I thought may be they are back.

And also I want to say that
> Old style miniplayer is available only for older versions,

This does not work, only first time you open the app it shows like old mini player layout and next time you open the app it changes the mini player layout (even in old version which I patched using revanced manager).

**Reply by anddea** [reply]
It should work for 6.40 and before. Doesn't it work?

**Reply by MrKK4** [reply]
No, I used youtube music version 6.41.58 (arm64-v8a) version and patched using revanced manager
When I open the app first time it shows the old layout and then I close the app and open again it changes the layout.

**Reply by anddea** [reply]
Can you show with screenshots (when you install and after restart the app)?

**Reply by MrKK4** [reply]
Actually the screenshots I gave earlier is the app that i patched 

# before restarting the app
![image](https://github.com/anddea/revanced-patches/assets/94269452/a46dd528-51e5-4421-9866-d6437dc293c7)


# After restarting the app

![image](https://github.com/anddea/revanced-patches/assets/94269452/0ad10bdd-aa8c-4c13-b288-bc1e7c16f3f0)

Do you need anything else.

**Reply by MrKK4** [reply]
I recently patched YT music version 6.40.53 but this also has the same problem 
after patching the first time open the app it shows old layout but as soon as I clear the app from backgrounds and open it again it changes the mini player layout.

**Reply by anddea** [reply]
Interesting, I have the same app version and it works fine, even if I restart it.

**Reply by MrKK4** [reply]
Is there any settings saved in phone for configuration
that might be causing the thing to happen.

**Reply by anddea** [reply]
No, I use default settings and patches. Do you remember what version was the last for this patch to work?

**Reply by MrKK4** [reply]
So, before I update my YT music version to before < 6.42 something it was working but after I used the latest versions I couldn't get it to work.

**Reply by MrKK4** [reply]
just to clarify is this the default settings you used for patching

![image](https://github.com/anddea/revanced-patches/assets/94269452/4eb6446d-d13d-4720-aee9-3214548eed0d)

**Reply by MrKK4** [reply]
#### How is this even working for you, I used My real device, Virtual master (android emulator), Nox Player (Android emulator for PC), all of them seems to have the same problem.

**Reply by anddea** [reply]
You didn't even include aforementioned patch - "Enable old style miniplayer". Those are not default patches, you have to click on "Selected patches" and click "Default" there. But "Enable old style miniplayer" is not included by default, you have to include it manually.

**Reply by MrKK4** [reply]
And where is that all i found is these patches

![image](https://github.com/anddea/revanced-patches/assets/94269452/26e9062c-995b-41da-b5c0-e59f1daef97d)


![image](https://github.com/anddea/revanced-patches/assets/94269452/5fc7f495-2fee-4aa5-8619-c45b6988497e)

**Reply by anddea** [reply]
You are using original ReVanced. You have to change sources in Manager settings by setting organization to anddea (or any other maintainer) for both Patches and Integrations.

**Reply by MrKK4** [reply]
Is this how it should be

![WhatsApp Image 2024-04-28 at 15 54 19_2ab89525](https://github.com/anddea/revanced-patches/assets/94269452/792e773a-9006-458d-bde2-4a96c5d06288)

**Reply by MrKK4** [reply]
Also just wanted to ask that after changing this (to anddea) the recommended app version is showing 6.33.52

![image](https://github.com/anddea/revanced-patches/assets/94269452/f72c7758-ba00-4a6b-bddb-910a2f6bf2cd)

**Reply by MrKK4** [reply]
Ok! so I've done the recommended settings and it seems it's working as expected.

Thank you very much, and Sorry! for the overhead.

**Reply by anddea** [reply]
6.33 has all patches working with no problems, but you can use newer versions if you don't need some patches that are obsolete for new YT Music versions.
